aspnetcore/toc.yml

```diff
@@ -746,6 +746,8 @@
       href: https://github.com/grpc/grpc-dotnet/tree/master/examples
     - name: Troubleshoot
       uid: grpc/troubleshoot
+    - name: Why migrate WCF to ASP.NET Core gRPC
```

Comment:
Move up to next to "Migrate gRPC services from C-core"

Reply:
Thanks.  Looks like I didn't get in an update for that.  I remember changing it.  Sloppy of me.

Reply:
...at first I thought I left a copy of troubleshoot in.  Will move it up.